### PR TITLE
qm-stop: add page

### PR DESCRIPTION
**Pull request by einverne**
#8537

- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/qm-stop.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Stop virtual machine immediately:
+
+`qm stop {{VM_ID}}`
+
+- Stop virtual machine and wait for at most 10 seconds:
+
+`qm stop --timeout {{10}} {{VM_ID}}`
+
+- Stop virtual machine and skip lock(only root can use this option):
```
Comment (kbdharun) [inline pages/common/qm-stop.md:14]:
```suggestion
- Stop virtual machine and skip lock (only root can use this option):
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. I have a small suggestion.

File: pages/common/qm-stop.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# qm stop
+
+> Stop virtual machine.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Stop virtual machine immediately:
```
Comment (CleanMachine1) [inline pages/common/qm-stop.md:6]:
```suggestion
- Stop a virtual machine immediately:
```

File: pages/common/qm-stop.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`qm stop {{VM_ID}}`
+
+- Stop virtual machine and wait for at most 10 seconds:
```
Comment (CleanMachine1) [inline pages/common/qm-stop.md:10]:
```suggestion
- Stop a virtual machine and wait for at most 10 seconds:
```

File: pages/common/qm-stop.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`qm stop --timeout {{10}} {{VM_ID}}`
+
+- Stop virtual machine and skip lock (only root can use this option):
```
Comment (CleanMachine1) [inline pages/common/qm-stop.md:14]:
```suggestion
- Stop a virtual machine and skip lock (only root can use this option):
```

File: pages/common/qm-stop.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`qm stop --skiplock {{true}} {{VM_ID}}`
+
+- Stop virtual machine and do not deactivate storage volumes:
```
Comment (CleanMachine1) [inline pages/common/qm-stop.md:18]:
```suggestion
- Stop a virtual machine and don't deactivate storage volumes:
```

File: pages/common/qm-stop.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# qm stop
+
+> Stop virtual machine.
```
Comment (CleanMachine1) [inline pages/common/qm-stop.md:3]:
```suggestion
> Stop a virtual machine.
```

**Review by CleanMachine1 — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**